### PR TITLE
doc/*: fix `k8s tainted` broken link

### PR DESCRIPTION
**Pull request by flyer103**
Sorry for another PR to fix broken link. 
I'm learning the etcd-operator's design to improve my operator project. The `doc/design` gives me much new thinking :D

[skip ci]

### CODE REVIEW

File: doc/design/cluster_status.md [hunk 1]
```diff
@@ -33,5 +33,5 @@ In summary, the TPR:
 
 - receive spec update (or initialize spec) with a resource version
 - collect cluster status during reconciliation
-- atomically update cluster status with known resource version after reconciliation if there is a status change
+- automatically update cluster status with known resource version after reconciliation if there is a status change
```
Comment (hongchaodeng) [inline doc/design/cluster_status.md]: Revert this change. It should be "atomic"

Comment (flyer103) [reply doc/design/cluster_status.md]: fixed

**Comment by hongchaodeng** [comment]
Thanks @flyer103
